Source/Core/Core/HW/WiimoteEmu/WiimoteEmu.cpp

```diff
@@ -259,7 +259,7 @@ Wiimote::Wiimote(const unsigned int index) : m_index(index), ir_sin(0), ir_cos(1
   for (const char* named_button : named_buttons)
   {
     const std::string& ui_name = (named_button == std::string("Home")) ? "HOME" : named_button;
-    m_buttons->controls.emplace_back(new ControllerEmu::Input(named_button, ui_name));
+    m_buttons->controls.emplace_back(new ControllerEmu::Input(false, named_button, ui_name));
```

Comment:
Same check as the START button above?

Reply:
No. You can read PR #5347 for the details, but the basic explanation is that we want START to be translatable because it's called スタート in Japanese GameCube games, whereas there's no such reason for HOME to be translated. START is marked with `_trans` in master and HOME isn't, so that I'm setting the translatability of HOME to `false` is consistent with what we're already doing.

Reply:
Ah, just looked up images of a JP Wii remote, and POWER and HOME are in English.